### ISSUE TITLE
Generation script for 3d front

### ISSUE THREAD

**Issue by HannahHaensen** [lyclyc52/NeRF_RPN#5]
Hey I am currently working also with blenderproc for a GT dataset for nerf. However, i have some problems with the transforms.json

https://github.com/DLR-RM/BlenderProc/issues/895

could you help me as you already did it for 3d front?

**Comment by lyclyc52** [comment]
We did not notice this kind of problem before... I think it is the problem of Blenderproc and Blender. Maybe you can calucate the camera angle by yourself using the focal length and the image size. 

BTW, will that small difference make great effect on your experiment? I think normally it will not make great impacts.

**Comment by HannahHaensen** [comment]
@lyclyc52 i agree however I have some weird Pose jittering with blenderproc. Would it be possible for you to share your front3d dataset script using blenderproc ?

**Comment by lyclyc52** [comment]
We put the [repo link](https://github.com/hjk0918/BlenderProc/tree/main/scripts) in this [README.md](https://github.com/lyclyc52/NeRF_RPN/tree/main/data)
